README.md

```diff
@@ -39,23 +43,84 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 - [Official Site](https://github.com/features/actions)
 - [Official Community Forum Board](https://github.community/t5/GitHub-Actions/bd-p/actions)
-- [Official Documentation](https://help.github.com/en/categories/automating-your-workflow-with-github-actions) (for YML format actions)
-- [Official Actions Collection](https://github.com/actions)
-- [GitHub Blog Announcement](https://blog.github.com/2018-10-17-action-demos/)
+- [Official Documentation](https://help.github.com/en/github/automating-your-workflow-with-github-actions)
+- [Official Actions organization](https://github.com/actions)
+  - [actions/virtual-environments](https://github.com/actions/virtual-environments) - GitHub Actions virtual environments.
+- [GitHub Blog Announcement](https://github.blog/2018-10-17-action-demos/)
 
-### Status badge
+### Workflow Examples
+
+- [actions/starter-workflows](https://github.com/actions/starter-workflows) - Starter workflow management.
+- [actions/example-services](https://github.com/actions/example-services) - Example workflows using service containers.
+
+### Official Actions
+
+#### Workflow Tool Actions
+
+Tool actions for your workflow.
+
+- [actions/checkout](https://github.com/actions/checkout) - Setup your repository on your workflow.
+- [actions/upload-artifact](https://github.com/actions/upload-artifact) - Upload artifacts from your workflow.
+- [actions/download-artifact](https://github.com/actions/download-artifact) - Download artifacts from your build.
+- [actions/cache](https://github.com/actions/cache) - Cache dependencies and build outputs in GitHub Actions.
+- [actions/github-script](https://github.com/actions/github-script) - Write a script for GitHub API and the workflow contexts.
+- [`actions/github`](https://github.com/actions/github) - Wraps actions-toolkit into an Action for common GitHub automations.
```

Comment:
Do you have another idea to skip this error?
https://github.com/sdras/awesome-actions/runs/286220732#step:4:8

Reply:
I'm not familiar with that tool, but isn't there any way to disable a specific rule for only one line?

Reply:
Linter is [sindresorhus/awesome-lint](https://github.com/sindresorhus/awesome-lint). There is no option like a ignore option.

Reply:
Ignore option is good for many people, so it's a good idea to create a PR and improve that package. At least I'm not very much in favor of a change that breaks the unity of the current style.

Reply:
By https://github.com/sindresorhus/awesome-lint/pull/78, we will be able to use the lint disable statement.

```md
<!--lint disable awesome-list-item-->
- [`actions/github`](https://github.com/actions/github) - Wraps actions-toolkit into an Action for common GitHub automations.
<!--lint enable awesome-list-item-->
```

Reply:
Go to #246